### ISSUE TITLE
fordpass_lastRefresh 1.54-Release2 time zone calculation issue?

### ISSUE THREAD

**Issue by rtw819** [itchannel/fordpass-ha#338]
fordpass_lastRefresh value shows "In 2 hours" (last refresh in the future?!) on the sensor summary page I have setup.  I suspect it is showing us the GMT time value, and not the localized time zone time.

Actual Current Local Time (at time of writing):  10/13 at 12:26am (EDT / GMT-4)

fordpass_lastRefresh - sensor detail
1 hour ago
Value:  October 13, 2023 at 2:41 AM

We got home and turned the vehicle off at just after 10 o'clock pm.  I have timed automations setup that kick off a poll request at 10pm and 10:30pm if the vehicle is in the "Home" zone. 

Per fordpass_gps sensor status, we returned home at:
UpdateTime            October 12, 2023 at 10:11:25 PM
OemCorrelationId   2633

Per the ignitionStatus sensor:

UpdateTime             October 12, 2023 at 10:36:15 PM
OemCorrelationId    2640
Value                        OFF

There's a 4 hour difference between 10pm and the 2am "next" lastRefresh.  That would seem to indicate GMT time for the sensor response, not our local EDT timezone value.  Let me know if there's anything I can do to help troubleshoot this?

**Comment by itchannel** [comment]
Yep it's actually a HA issue. For some reason they covert the local timestamp back to GMT when showing the sensor.

Someone raised a issue on the official HA GitHub 

https://github.com/home-assistant/core/issues/101371
